main.py

```diff
@@ -63,7 +63,11 @@ class File(BaseModel):
 app = FastAPI()
 
 @app.get("/users")
-async def get_users():
+async def get_users(limit: int = 0):
+  if limit < 0:
+      raise HTTPException(status_code=400, detail="'limit' must be greater than zero")
```

Comment:
Nit: "limit must be non-negative" (alternatively, "limit cannot be negative").

Reply:
Thanks, fixed in a565331.